.storybook/playground/index.js

```diff
@@ -24,7 +24,7 @@ import '@wordpress/format-library';
 /**
  * Internal dependencies
  */
-import './style.scss';
+// import './style.scss';
```

Comment:
@simison I'd appreciate your help here. I couldn't make it work with `@wordpress/base-styles` which are imported in SCSS files. I didn't go very far with debugging. It might be even the case that Storybook doesn't support SCSS out of the box.

Reply:
> It might be even the case that Storybook doesn't support SCSS out of the box.

That's it. Storybook only supports `.css`.

Adding `.scss` support should be relatively straight forward. (Unless we have something funky with our `.scss` builds)

https://storybook.js.org/docs/configurations/custom-webpack-config/

Reply:
Awesome work 🥇